apps/emqx/src/emqx_hookpoints.erl

```diff
@@ -59,6 +59,7 @@
     'message.publish',
     'message.puback',
     'message.dropped',
+    'message.validation_failure',
```

Comment:
maybe beter to follow the majority verb style naming.
```suggestion
    'message.validation_failed',
```

Reply:
I followed the EIP: https://github.com/emqx/eip/blob/c898b13394160f0694e75a3fc5e7db9768377aa3/active/0024-message-validation.md?plain=1#L122